.eslintrc.json

```diff
@@ -38,7 +38,7 @@
   // Do NOT change these rules
   "rules": {
     "indent": [2, 2, {"SwitchCase": 1}],
-    "max-len": [2, 80, 4, {
+    "max-len": [2, 100, 4, {
```

Comment:
Not today's problem, but we should update this package to use our eslint rules from alley-scripts

Reply:
It's on my list of TODO.